### PR TITLE
Updated Delete FAB icon (#2682)

### PR DESCRIPTION
**Pull request by vince-nyanga**
Screenshot:

screenshot_1528221195

### CODE REVIEW

File: app/src/main/res/drawable/ic_delete.xml [hunk 1]
```diff
@@ -1,9 +1,7 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="24dp"
-        android:height="24dp"
-        android:viewportWidth="24.0"
-        android:viewportHeight="24.0">
-    <path
-        android:pathData="M6,19c0,1.1 0.9,2 2,2h8c1.1,0 2,-0.9 2,-2V7H6v12zM19,4h-3.5l-1,-1h-5l-1,1H5v2h14V4z"
-        android:fillColor="#ffffff"/>
+<vector android:height="24dp" android:viewportHeight="16"
```
Comment (ekager) [inline app/src/main/res/drawable/ic_delete.xml]: Could you add a license header? You can copy it from the top of any of the other vector files

File: app/src/main/res/drawable/ic_delete.xml [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
+    android:viewportWidth="16" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#ffffff"
```
Comment (ekager) [inline app/src/main/res/drawable/ic_delete.xml]: I commented in #2682 with new path data that is more centered, not sure why this svg was off, but it was our fault, not yours. Thanks!

Comment (vince-nyanga) [reply app/src/main/res/drawable/ic_delete.xml]: I've added it.

**Comment by vince-nyanga** [comment]
New screenshot:
capture

**Comment by aminalhazwani** [comment]
@ekager I redraw the delete icon to make it possible to align it to the center without hacks. The icon itself is is 18x20dp but the SVG includes a 24x24dp bounding box as per Material specs. This would allow you to easily center the icon in a 56x56dp FAB.

screen shot 2018-06-08 at 3 18 44 pm

```
<!--This Source Code Form is subject to the terms of the Mozilla Public - License, v. 2.0. If a copy of the MPL was not distributed with this - file, You can obtain one at http://mozilla.org/MPL/2.0/.-->
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path fill-rule="evenodd" d="M20 5h-5a3 3 0 1 0-6 0H4a1 1 0 0 0 0 2h1v12a3 3 0 0 0 3 3h8a3 3 0 0 0 3-3V7h1a1 1 0 1 0 0-2zm-8-2a2 2 0 0 1 2 2h-4a2 2 0 0 1 2-2zM7 7v12a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V7H7zm2.539 2.002a.5.5 0 0 0-.503.498l-.034 8a.5.5 0 0 0 1 .004l.034-8a.5.5 0 0 0-.497-.502zm4.497.498a.5.5 0 0 1 1 .004l-.034 8a.5.5 0 0 1-1-.004l.034-8zM12 9.002a.5.5 0 0 0-.502.498l-.034 8a.5.5 0 0 0 1 .004l.034-8A.5.5 0 0 0 12 9.002z" fill="#000"/>
</svg>
```

**Comment by aminalhazwani** [comment]
jfyi I am in the process or redrawing all the Android icons, for both mobile and TV. Initially the icons were scaled up from the 16x16px Desktop icons but this is not ideal. I am redrawing everything with a live area of 20x20dp surrounded by a bounding box of 24x24dp 🎉

**Comment by aminalhazwani** [comment]
Sure @ekager, here the file [delete-24.zip](https://github.com/mozilla-mobile/focus-android/files/2095726/delete-24.zip) 🙂

**Comment by ekager** [comment]
Thanks @aminalhazwani ! @vince-nyanga do you want to import the new icon linked above? Thanks for your patience and help!

**Comment by vince-nyanga** [comment]
@ekager I'll update it ASAP.

**Comment by vince-nyanga** [comment]
This is how it looks:
capture

**Comment by ekager** [comment]
Hi @brampitoyo what do you think about the new centered svg above?

**Comment by brampitoyo** [comment]
@ekager The centred svg looks really good! Thanks for posting the screenshot, @vince-nyanga.

@aminalhazwani Your redrawing plan sounds good. I’d love to help! When it’s finished, replacing our assets may be a simple matter of us pushing a few commits into this repo.

**Pull request merged**